liquidprompt

```diff
@@ -1695,6 +1698,38 @@ else
     _lp_time() { : ; }
 fi
 
+##################################
+# Detect if we're in a container #
+##################################
+
+_lp_container()
+{
+    (( LP_ENABLE_CONTAINER )) ||  return
+
+    _lp_container=""   
+
+    if [[ "$OSTYPE" == "linux-gnu" ]] ; then 
+        if test -e /.dockerenv ||  grep -q "docker" /proc/1/cgroup  ; then
```

Comment:
I have tested these checks myself and they work, but I would like to see links to documentation on this in the commit messages somewhere.